Cargo.toml

```diff
@@ -50,11 +49,16 @@ serde_json = "1.0"
 sha2 = "0.10.8"
 thiserror = "1.0"
 tls_codec = "0.4.0"
-tokio = { version = "1.35.1", features = ["macros"] }
-tonic = "^0.11"
-tracing = { version = "0.1", features = ["release_max_level_debug"] }
+tokio = { version = "1.35.1", default-features = false }
+async-stream = "0.3"
+tracing = { version = "0.1" }
 tracing-subscriber = "0.3"
 url = "2.5.0"
+tonic = "^0.11" 
+prost = "^0.12"
+prost-types = "^0.12"
+pbjson = "0.5.1"
```

Comment:
Have we run into any cases where `pbjson` doesn't serialize the same way that GRPC gateway expects? Or is it guaranteed to be 1:1?

Reply:
Looked at this again with a fresh head, this was an issue trying to update `pbjson` to `0.7`, but making everything point to `0.6` works. Updating `pbjson` to 0.7 forces an update for `prost` & `tonic` too, because of trait implementations. But updating those dependencies results in test failures because of TLS problems which can be tackled separately.